Implemented mising blending filters

Shiwank19 opened the pull request:
I have researched missing blending filter formulae and implemented them.Please have a look.

abhikpal commented:
The code looks okay to me. But before I test things out, I have comments:

1. Could you do a merge request against  the `develop` branch? 
2. It would be nice to have some examples to go with the code. Add them here: (https://github.com/p5py/p5-examples). I usually use the following convention with examples:
   1. if it's from the examples section from Processing (Java) (https://processing.org/examples/) then it goes in the folder with the appropriate section name (so `basics/math/additive_wave.py` is a port of https://processing.org/examples/additivewave.html)
    2. If it's from the reference section, it goes in the reference folder (with a similar folder structure as (i))

Shiwank19 commented:
ok I am working on both the points you mentioned.
Sir My code is related to image directory inside examples.I am going to add examples for each blend mode.

Shiwank19 commented:
I have made pull request to develop branch of p5py and also added new examples for each blending mode I've implemented.I have created a [pull request to develop branch of p5-examples](https://github.com/p5py/p5-examples/pull/6).
Please review it.
